media-libs/sdl_pango/sdl_pango-0.1.2.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="This library is a wrapper around the Pango library"
+DESCRIPTION="SDL_Pango library allows you to use TrueType fonts to \
+render internationalized and tagged text in SDL applications."
+HOMEPAGE="http://www.libsdl.org/projects/SDL_pango/"
+COPYRIGHT="2004 NAKAMURA Ken'ichi'"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/sdlpango/files/SDL_Pango/0.1.2/SDL_Pango-$portVersion.tar.gz"
+CHECKSUM_SHA256="7f75d3b97acf707c696ea126424906204ebfa07660162de925173cdd0257eba4"
+SOURCE_DIR="SDL_Pango-$portVersion"
+PATCHES="sdl_pango-$portVersion.patchset"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	sdl_pango$secondaryArchSuffix = $portVersion
+	lib:libSDL_Pango$secondaryArchSuffix = 1.1.0 compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libSDL$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	#needed for pango
+	lib:libglib_2.0$secondaryArchSuffix
+	#needed for pangoft2
+	lib:libgraphite2$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	sdl_pango${secondaryArchSuffix}_devel = $portVersion
+	devel:libSDL_Pango$secondaryArchSuffix = 1.1.0 compat >= 1
+	"
+REQUIRES_devel="
+	sdl_pango$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libSDL$secondaryArchSuffix
+	devel:libpango_1.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	#needed for pango
+	lib:libglib_2.0$secondaryArchSuffix
+	#needed for pangoft2
+	devel:libgraphite2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtool$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autoreconf -fi
+	runConfigure ./configure --prefix=$prefix
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	# remove libtool library file
+	rm $libDir/libSDL_Pango.la
+
+	prepareInstalledDevelLibs libSDL_Pango
```

Comment:
You can use prepareInstalledDevel**Lib** instead of prepareInstalledDevel**Libs** because we only have one lib here.